components/camel-mllp/pom.xml

```diff
@@ -28,6 +28,7 @@
     </parent>
 
     <artifactId>camel-mllp</artifactId>
+<!--    <version>4.9.0</version>-->
```

Comment:
This should be a snapshot

Reply:
Ah yes thats from an earlier commit i forgot to remove.